src/styles/blocks/stars--3.css

```diff
@@ -0,0 +1,9 @@
+.stars--3 .stars__star:nth-child(-n + 3) {
+  background-image: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

Comment:
Avoid using tag names for styling. It's better to style elements using classes to prevent potential conflicts and improve CSS specificity. Consider refactoring the selector to use a class instead of the `nth-child` pseudo-class if possible, or ensure that the usage of `nth-child` here is absolutely necessary and cannot be replaced with a class.